server/routerlicious/packages/services-core/src/runWithRetry.ts

```diff
@@ -7,6 +7,19 @@ import { delay } from "@fluidframework/common-utils";
 import { Lumber, LumberEventName, Lumberjack } from "@fluidframework/server-services-telemetry";
 import { NetworkError } from "@fluidframework/server-services-client";
 
+interface runWithRetryParams<T> {
```

Comment:
Should we do the same for `requestWithRetry`, defined below `runWithRetry`? If yes, we might also want to extract the common parameters into a "base" interface so we can reuse it

Reply:
Yes, we should do it for `requestWithRetry` as well.. I didnt want to include that in the same PR so that it can be scoped to only scriptorium fixes. But I can add a TODO comment there so we can pick it up in another PR.

Reply:
Resolving as I removed the changes for runWithRetry and will follow up in another PR